src/handle-data.ts

```diff
@@ -65,11 +65,13 @@ export function getColumnInfo(layout: TableLayout, colIdx: number, pageColIdx: n
   const { qError, qFallbackTitle, textAlign, qAttrExprInfo, qSortIndicator, qReverseSort, qApprMaxGlyphCount } = info;
   const isHidden = qError?.qErrorCode === 7005;
   const isLocked = isDim && (info as ExtendedNxDimensionInfo).qLocked;
+  const isMasterDim = isDim && !!(info as ExtendedNxDimensionInfo).qLibraryId;
```

Comment:
I think it makes more sense to check this for both measure and dimension
```suggestion
  const isMasterItem = !!info.qLibraryId;
```

Reply:
Does master measure has qLibId as well? i get type complains about it if i do so